core/src/main/java/io/neow3j/protocol/core/JsonRpc2_0Neow3j.java

```diff
@@ -1051,6 +1052,37 @@ public Request<?, NeoSendToAddress> sendToAddress(TransactionSendToken txSendTok
                 txSendToken.getValue());
     }
 
+    /**
+     * Attempt to cancel a transaction that has been sent to the network if it is not yet confirmed. This is achieved by
+     * creating a conflicting transaction (using the 'conflicts' transaction attribute). The conflicting transaction
+     * has a minimally higher network fee than the transaction to cancel. You can also add an extra network fee to boost
+     * priority. Make sure that the involved signers are the same as in the transaction to cancel. The first signer will
+     * be used to pay for GAS costs. Signers can also be contracts that have a verify method.
```

Comment:
Do the signers need to be in the same order as well?

If so, please consider adding it here and potentially also in the `@param` javadoc entry, e.g.,:

```suggestion
     * priority. Make sure the involved signers are the same and are in the same order as in the transaction to cancel. 
     * The first signer will be used to pay for GAS costs. Signers can also be contracts that have a verify method.

Reply:
Checked with Anna and the core code: The signers don't need to be in the same order as the 'original' transaction. Not even all of the signers need to be there. They just have to overlap with the set of signers on the 'original' transaction.

Adapting the javadoc.